optuna/integration/allennlp.py

```diff
@@ -157,22 +328,47 @@ class AllenNLPPruningCallback(EpochCallback):
     if you want to add a proning callback which observes a metric.
 
     Args:
-        trial:
-            A :class:`~optuna.trial.Trial` corresponding to the current evaluation of the
-            objective function.
         monitor:
             An evaluation metric for pruning, e.g. ``validation_loss`` or
             ``validation_accuracy``.
+        trial:
+            A :class:`~optuna.trial.Trial` corresponding to the current evaluation of the
+            objective function.
```

Comment:
`trial` should be put top of the `Args` block because the order of arguments of dunder init is trial and monitor.

Reply:
You're right, thank you! v